expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -118,6 +128,15 @@ public ExpandedDataEntry expandEntry(Entity dataEntry) throws JsonProcessingExce
         throw new UnsupportedOperationException(UNSUPPORTED_TYPE + dataEntry.getClass().getSimpleName());
     }
 
+    private ObjectNode replaceInlineContextWithUriContext(String resourceWithNviValues) {
+        var objectNode =  attempt(() -> (ObjectNode) objectMapper.readTree(resourceWithNviValues)).orElseThrow();
+        if (!objectNode.at(CONTEXT_NODE_SELECTOR).isMissingNode()) {
+            objectNode.remove(CONTEXT_FIELD_PROPERTY_NAME);
+            objectNode.put(CONTEXT_FIELD_PROPERTY_NAME, CONTEXT_URI);
```

Comment:
.put replaced content, so it should be unecasery to remove() first

Reply:
Ha, I didn't think it did, but apparently so!